core/src/it/units/sdm/project/game/gui/GameScreen.java

```diff
@@ -0,0 +1,247 @@
+package it.units.sdm.project.game.gui;
+
+import com.badlogic.gdx.*;
+import com.badlogic.gdx.graphics.*;
+import com.badlogic.gdx.graphics.g2d.Sprite;
+import com.badlogic.gdx.maps.tiled.TiledMap;
+import com.badlogic.gdx.maps.tiled.TiledMapTileLayer;
+import com.badlogic.gdx.maps.tiled.renderers.OrthogonalTiledMapRenderer;
+import com.badlogic.gdx.maps.tiled.tiles.StaticTiledMapTile;
+import com.badlogic.gdx.scenes.scene2d.Actor;
+import com.badlogic.gdx.scenes.scene2d.EventListener;
+import com.badlogic.gdx.scenes.scene2d.InputEvent;
+import com.badlogic.gdx.scenes.scene2d.Stage;
+import com.badlogic.gdx.scenes.scene2d.utils.ClickListener;
+import com.badlogic.gdx.utils.ScreenUtils;
+import it.units.sdm.project.board.gui.GuiBoard;
+import it.units.sdm.project.board.Position;
+import it.units.sdm.project.board.gui.GuiStone;
+import it.units.sdm.project.enums.GameStatus;
+import it.units.sdm.project.game.FreedomPointsCounter;
+import it.units.sdm.project.game.Move;
+import it.units.sdm.project.game.Player;
+import org.jetbrains.annotations.NotNull;
+import org.jetbrains.annotations.Nullable;
+
+import java.util.LinkedList;
+import java.util.NoSuchElementException;
+import java.util.Set;
+
+public class GameScreen implements Screen {
+
+    private static final int SQUARE_WIDTH_IN_PIXELS = 32;
+
+    private static final int NUMBER_OF_ROWS = 8;
+    private static final int NUMBER_OF_COLUMNS = 8;
+    @NotNull
+    private GameStatus gameStatus;
+    @NotNull
+    private final LinkedList<Move> playersMovesHistory = new LinkedList<>();
+    @NotNull
+    private final OrthographicCamera camera;
+    @NotNull
+    private final OrthogonalTiledMapRenderer render;
+    @NotNull
+    private final Stage stage;
+    @NotNull
+    private final GuiBoard board;
+    @NotNull
+    private final Texture blackStoneImage = new Texture(Gdx.files.internal("./assets/circle2.png"));
```

Comment:
Still same problem as mentioned in #83. You either need to change your run configuration or we need to change the configuration in the `build.gradle` files.